openlibrary/templates/lists/lists.html

```diff
@@ -34,11 +34,12 @@
                                         var list_id = "#" + \$(this).data("list-id");
                                         var key = \$(list_id + " a:first").attr("href");
                                         var dialog = this;
-
+            
                                         \$.post(key + "/delete.json", function() {
                                             \$(list_id).remove();
                                             \$(dialog).dialog("close");
-                                        });
+                                            \$("#list_count").load(location.href + " #list_count ");
```

Comment:
Well that's cool! TIL jquery `.load` :)
```suggestion
                                            \$("#list_count").load(location.href + " #list_count");
```